index.html

```diff
@@ -23,14 +23,18 @@ <h3 class="ribbon-hero-description-light">
 <section class="slab-alpha container-padded explore_home-map">
   <div class="container-page-wrapper">
     <div class="container-left-4 explore_home-map-intro">
-      <p class="h5 explore_home-subhead">Explore data</p>
       <h2 class="h3 explore_home-heading">Learn about natural resource extraction in each state</h2>
       <p>
         Explore production, revenue, and economic impact data for each state, as well as additional contextual information and data about three states that chose to participate in deeper USEITI reporting: <a href="{{ site.baseurl }}/explore/AK/">Alaska</a>, <a href="{{ site.baseurl }}/explore/MT/">Montana</a>, and <a href="{{ site.baseurl }}/explore/WY/">Wyoming</a>.
       </p>
       <figcaption class="state_pages-select">
-        <label for="location-selector">Choose a state or offshore region</label>
-        {% include selector.html %}
+        <label for="location-selector">Explore data</label>
+        {% include selector.html
+           nationwide = true
+           nationwide_url = '/explore/'
+           nationwide_title = 'Nationwide data'
+           default = 'Choose one (COREY???)'
```

Comment:
@coreycaitlin we were struggling to pick a default set of text here. To be consistent with the previous layout, we would want `Choose a state or offshore region`, but the width of the selector doesn't fully accommodate that text. Any ideas for how to succinctly communicate what people can select?